dask_jobqueue/core.py

```diff
@@ -88,6 +91,11 @@ class JobQueueCluster(Cluster):
     cancel_command = None
     scheduler_name = ''
 
+    # Required for excuting commands through the shell in the subprocess module.
+    # It handles the shell input redirection e.g. bsub < script_filename.sh
+    # and does not consider '<' as a command, file or directory.
+    shell = False
```

Comment:
Personally I would not use a comment in the base class.

Also I would go for a more informative variable name, maybe popen_shell = True (better name more than welcome)?